File: CursoCsharpUdemy/exer/outros/Exer37.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exer.outros
{
    public class Exer37
    {
        static void Main(string[] args)
        {
            int operation = 0;

            do
            {
                Console.WriteLine("Selecione a operação desejada:");
                Console.WriteLine("1 - Adição");
                Console.WriteLine("2 - Subtração");
                Console.WriteLine("3 - Multiplicação");
                Console.WriteLine("4 - Divisão");
                Console.WriteLine("5 - Sair");

                operation = int.Parse(Console.ReadLine());

                switch (operation)
                {
                    case 1:
                        Console.Write("Digite o primeiro número: ");
                        double num1 = double.Parse(Console.ReadLine());

                        Console.Write("Digite o segundo número: ");
                        double num2 = double.Parse(Console.ReadLine());

                        double resultado = num1 + num2;

                        Console.WriteLine("Resultado: " + resultado);
                        break;

                    case 2:
                        Console.Write("Digite o primeiro número: ");
                        num1 = double.Parse(Console.ReadLine());

                        Console.Write("Digite o segundo número: ");
                        num2 = double.Parse(Console.ReadLine());

                        resultado = num1 - num2;

                        Console.WriteLine("Resultado: " + resultado);
                        break;

                    case 3:
                        Console.Write("Digite o primeiro número: ");
                        num1 = double.Parse(Console.ReadLine());

                        Console.Write("Digite o segundo número: ");
                        num2 = double.Parse(Console.ReadLine());

                        resultado = num1 * num2;

                        Console.WriteLine("Resultado: " + resultado);
                        break;

                    case 4:
                        Console.Write("Digite o primeiro número: ");
                        num1 = double.Parse(Console.ReadLine());

                        Console.Write("Digite o segundo número: ");
                        num2 = double.Parse(Console.ReadLine());

                        if (num2 == 0)
                        {
                            Console.WriteLine("Não é possível dividir por zero!");
                        }
                        else
                        {
                            resultado = num1 / num2;
                            Console.WriteLine("Resultado: " + resultado);
                        }
                        break;

                    case 5:
                        Console.WriteLine("Encerrando o programa...");
                        break;

                    default:
                        Console.WriteLine("Opção inválida! Tente novamente.");
                        break;
                }

                Console.WriteLine();

            } while (operation != 5);
        }
    }
}

File: Curso de Java/Exer/src/lista64Exer/Exer37.java

/*******************************************************************************
 * 37) Faça um programa de conversão de base numérica. O programa deverá
 * apresentar uma tela de entrada com as seguintes opções:
 *		1 – Adição
 *		2 – Subtração
 *		3 – Multiplicação
 *		4 – Divisão
 * Informe a opção:
 * A partir da opção escolhida, o programa deverá solicitar para que o usuário
 * digite dois números. Em seguida, o programa deve exibir o resultado da opção
 * indicada pelo usuário e perguntar ao usuário se ele deseja voltar ao menu
 * principal. Caso a resposta seja  ́S ́ ou  ́s ́, deverá voltar ao menu, caso
 * contrário deverá encerrar o programa.
 *******************************************************************************/

import java.util.Scanner;

public class Exer37 {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int operation = 0;

        do {
            System.out.println("Selecione a operação desejada:");
            System.out.println("1 - Adição");
            System.out.println("2 - Subtração");
            System.out.println("3 - Multiplicação");
            System.out.println("4 - Divisão");
            System.out.println("5 - Sair");

            operation = sc.nextInt();

            if (operation == 1) {
                System.out.print("Digite o primeiro número: ");
                double num1 = sc.nextDouble();

                System.out.print("Digite o segundo número: ");
                double num2 = sc.nextDouble();

                double resultado = num1 + num2;

                System.out.println("Resultado: " + resultado);
            } else if (operation == 2) {
                System.out.print("Digite o primeiro número: ");
                double num1 = sc.nextDouble();

                System.out.print("Digite o segundo número: ");
                double num2 = sc.nextDouble();

                double resultado = num1 - num2;

                System.out.println("Resultado: " + resultado);
            } else if (operation == 3) {
                System.out.print("Digite o primeiro número: ");
                double num1 = sc.nextDouble();

                System.out.print("Digite o segundo número: ");
                double num2 = sc.nextDouble();

                double resultado = num1 * num2;

                System.out.println("Resultado: " + resultado);
            } else if (operation == 4) {
                System.out.print("Digite o primeiro número: ");
                double num1 = sc.nextDouble();

                System.out.print("Digite o segundo número: ");
                double num2 = sc.nextDouble();

                if (num2 == 0) {
                    System.out.println("Não é possível dividir por zero!");
                } else {
                    double resultado = num1 / num2;
                    System.out.println("Resultado: " + resultado);
                }
            } else if (operation == 5) {
                System.out.println("Encerrando o programa...");
            } else {
                System.out.println("Opção inválida! Tente novamente.");
            }

            System.out.println();

        } while (operation != 5);

        sc.close();

    }

}
